packages/discovery-provider/src/api/v1/users.py

```diff
@@ -2825,3 +2826,46 @@ def _get(self, id, authed_user_id):
     @auth_middleware(user_feed_parser, require_auth=True)
     def get(self, id, authed_user_id):
         return self._get(id, authed_user_id)
+
+
+muted_users_route_parser = reqparse.RequestParser(argument_class=DescriptiveArgument)
+
+MUTED_USERS_ROUTE = "/<string:id>/muted"
+
+
+@ns.route(MUTED_USERS_ROUTE)
```

Comment:
good to know about default and full, should i do that with the endpoints im adding?

Reply:
from my understanding it should be 1:1 ideally